hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/hudi/functional/cdc/TestCDCDataFrameSuite.scala

```diff
@@ -228,7 +227,8 @@ class TestCDCDataFrameSuite extends HoodieCDCTestBase {
   def testMORDataSourceWrite(cdcSupplementalLoggingMode: String): Unit = {
     val options = commonOpts ++ Map(
       DataSourceWriteOptions.TABLE_TYPE.key() -> DataSourceWriteOptions.MOR_TABLE_TYPE_OPT_VAL,
-      HoodieTableConfig.CDC_SUPPLEMENTAL_LOGGING_MODE.key -> cdcSupplementalLoggingMode
+      HoodieTableConfig.CDC_SUPPLEMENTAL_LOGGING_MODE.key -> cdcSupplementalLoggingMode,
+      HoodieCleanConfig.AUTO_CLEAN.key -> "false"
```

Comment:
why need to set `HoodieCleanConfig.AUTO_CLEAN` here. This config has been set in `commonOpts`.